lib/screen_controller.dart

```diff
@@ -496,6 +498,18 @@ class ScreenController {
         evaluateExtraContext(action.title),
         evaluateExtraContext(action.body),
       );
+    } else if (action is InitNotificationAction) {
+      final isEnabled = await notificationUtils.initNotifications() ?? false;
+
+      if (isEnabled && action.onAccept != null) {
+        executeAction(context, action.onAccept!);
+      }
+
+      if (!isEnabled && action.onReject != null) {
+        executeAction(context, action.onReject!);
+      }
+
+      if (action.onComplete != null) executeAction(context, action.onComplete!);
```

Comment:
what is the onComplete for if you already have onAccept or on Reject?

Reply:
yes, removed